lamindb_setup/_check_setup.py

```diff
@@ -70,14 +71,48 @@ def _get_current_instance_settings() -> InstanceSettings | None:
         return None
 
 
+def _check_in_modules(module: str, isettings: InstanceSettings | None = None) -> None:
+    not_in_instance_msg = f"'{module}' is missing from this instance."
+
+    if isettings is not None:
+        if module not in isettings.modules:
+            raise RuntimeError(not_in_instance_msg)
+        else:
+            return
+
+    from django.apps import apps
+
+    for app in apps.get_app_configs():
+        if module == app.name:
+            return
+    raise RuntimeError(not_in_instance_msg)
+
+
+def _module_name() -> str | None:
+    stack = inspect.stack()
+    if len(stack) < 3:
+        return None
+    module = inspect.getmodule(stack[2][0])
+    return module.__name__.partition(".")[0] if module is not None else None
+
+
 # we make this a private function because in all the places it's used,
 # users should not see it
 def _check_instance_setup(from_module: str | None = None) -> bool:
-    if django.IS_SETUP:
+    if django_lamin.IS_SETUP:
         # reload logic here because module might not yet have been imported
         # upon first setup
-        if from_module is not None and from_module != "lamindb":
-            il.reload(il.import_module(from_module))
+        if from_module is not None:
+            if from_module != "lamindb":
+                _check_in_modules(from_module)
+                il.reload(il.import_module(from_module))
+        else:
+            infer_module = _module_name()
```

Comment:
Why is this needed?

Reply:
What exactly?
this gets triggered for example in this case:
import lamindb as ln # connected, django is setup
import wetlab as wl # calls _check_instance_setup() without specifying from_module, module name needs to be inferred.